GitTfs.Vs2010/GitTfs.Vs2010.csproj

```diff
@@ -1,4 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Changed 2017 by Sijmen J. Mulder <s.mulder@betabit.nl>:
+   - fix xcopy invocation for paths with spaces
+-->
```

Comment:
Can you remove this note? Git's history will reveal this, if necessary. We don't have other similar notes inline, and prefer include this information in the release notes.

Reply:
I would rather not have included it in the first place but the license plainly requires it:

>      (b) You must cause any modified files to carry prominent notices
>          stating that You changed the files; and

Reply:
That's if you were redistributing this code on your own, not the main source code repository

Reply:
Then I can't modify my PR branch. Feel free to strip it off here though.